generate_workspace/src/main/java/com/google/devtools/build/workspace/output/BzlWriter.java

```diff
@@ -32,27 +35,33 @@
       MethodHandles.lookup().lookupClass().getName());
 
   private final String[] argv;
-  private final Path generatedFile;
+  private final File generatedFile;
```

Comment:
> Hmm. There's not much point of using jimfs if it won't let us use an in-memory filesystem anyway. Please modify this to either be all in memory or all on disk (for tests).

Wait, check the follow up comment. using `path.toFile()` didn't use the in-memory filesystem. We needed the absolute path. From what I understand, using `file.getAbsoluteFile()` let us use the in-memory file system. Each of the files were written to the bazel-sandbox's _tmp subdirectory, and were deleted after test completion.